### ISSUE TITLE
AWizard accessibility test 9.1.3 Anpassbar

### ISSUE THREAD

**Issue by ilia-brykin** [ilia-brykin/aloha#141]
AWizard accessibility test 9.1.3 Anpassbar
https://bitvtest.de/pruefverfahren/bitv-20-web

**Comment by babeyeva** [comment]
**Prüfschritt 9.1.3.1a**
This level of the text should have **h3**
![AWizard 9 1 3 1a](https://github.com/user-attachments/assets/b299097b-6f12-477a-af45-f2088fb2e317)

**Prüfschritt 9.1.3.1b** corresponds.

**Prüfschritt 9.1.3.1c** is not applicable for AWizard.

**Prüfschritt 9.1.3.1d** corresponds.

**Prüfschritt 9.1.3.1e** is not applicable for AWizard.

**Prüfschritt 9.1.3.1f** is not applicable for AWizard.

**Prüfschritt 9.1.3.1d** corresponds (But i'm not sure, i need advice.

![AWizard 9 1 3 1g 2](https://github.com/user-attachments/assets/f59cd2bf-b568-4c98-94b5-4fd39d7a54c8)
![AWizard 9 1 3 1g 1](https://github.com/user-attachments/assets/c09b5629-54eb-4f41-9320-5bb3f0fb38c3)

**Prüfschritt 9.1.3.1h** is not applicable for AWizard.

**Prüfschritt 9.1.3.2**

**Comment by ilia-brykin** [comment]
- [x] Prüfschritt 9.1.3.1a 
Prüfschritt 9.1.3.1d It's good that you found this, but it's about something else. It refers to old approaches to page organization.

**Comment by ilia-brykin** [comment]
Prüfschritt 9.1.3.3 There are no references to sensory characteristics here. What you mean is another step.

**Comment by babeyeva** [comment]
Corrected.
